neural_search/similarity_threshold.py

```diff
@@ -63,8 +69,17 @@ def get_scores(
         Returns:
             List[float]: containing the pairwise distances
         """
+        embedding_item = embedding.get(project_id, embedding_id)
+        if (
+            embedding_item.platform == EmbeddingPlatform.PYTHON.value
+            and embedding_item.model == "tf-idf"
+        ):
+            # tf idf embeddings are very similar by default as usually the vectors have a lot of 0s and only very few filled values => threshold doesn't make sense
+            return NO_THRESHOLD_INDICATOR
```

Comment:
I would expect this to be problematic for the np.percentile calculations, if we use  the NO_THRESHOLD_INDICATOR as a return value (not a List[float])

Reply:
Changed to list return . FYI never used for calculations only as indicator so not really an values that is used in np.percentile